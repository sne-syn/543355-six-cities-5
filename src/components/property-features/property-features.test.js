import React from 'react';
import renderer from 'react-test-renderer';
import PropertyFeatures from './property-features';

const accomodationType = `room`;

test(`Render PropertyFeatures`, () => {
  const tree = renderer
    .create(<PropertyFeatures
      type={accomodationType}
      bedrooms={3}
      maxGuests={2}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
