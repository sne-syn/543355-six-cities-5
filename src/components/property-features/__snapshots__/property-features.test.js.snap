// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PropertyFeatures 1`] = `
<ul
  className="property__features"
>
  <li
    className="property__feature property__feature--entire"
  >
    Private Room
  </li>
  <li
    className="property__feature property__feature--bedrooms"
  >
    3
     Bedrooms
  </li>
  <li
    className="property__feature property__feature--adults"
  >
    Max 
    2
     adults
  </li>
</ul>
`;
